README.md

```diff
@@ -56,6 +54,44 @@ brew install foundry
 
 Run `foundry version` to verify it's installed correctly.
 
+## Getting Started
+
+Clone this sample to your local system, or [download as a zip file](https://github.com/CrowdStrike/foundry-sample-scalable-rtr/archive/refs/heads/main.zip).
+
+```shell
+git clone https://github.com/CrowdStrike/foundry-sample-scalable-rtr
+cd foundry-sample-scalable-rtr
+```
+
+Log in to Foundry:
+
+```shell
+foundry login
+```
+
+Select the following permissions:
+
+- [x] Run RTR Scripts
+- [x] Run, execute, and test Workflows
+- [x] Run, execute, and test API integrations
+- [x] Run, execute, and test LogScale queries
+- [ ] (optional) Generate mock data to test your app
+
+Deploy the app to Foundry:
+
+```shell
+foundry apps deploy
+```
+
+> [!TIP]
+> If you get an error that the name already exists, change the name to something unique to your CID in `manifest.yml`.
+
+Once the deployment has finished, release the app using the three-dot menu on the right. Next, go to **Foundry** > **App catalog**, find your app, and install it.
```

Comment:
@hjabbott It seems like "three-dot menu on the right" might not be accessible. Would you happen to have a suggestion for improving this sentence?

Reply:
Great question - however, given that there's an MI underway to change the UI for the app lifecycle flow (Simplified Wayfinding), it would be better to future-proof by removing detailed UI instructions from this readme. The Deploy/Release/Install flow will soon be even more obvious in the UI than it is now.

It also just occurred to me that the readme suddenly jumps from the CLI to the UI without explicitly stating this. Do we want to keep users on the CLI in these readmes, or state that deployment errors aren't shown on the CLI so it's better to continue in the UI?

I realize I've been reviewing your changes bit by bit instead of reading the whole readme, and I need to take some time to review the whole file.

Reply:
> The Deploy/Release/Install flow will soon be even more obvious in the UI than it is now.

That's good to know. I feel like it's kind of painful now because you to deploy and enter release notes. Then, you have to release it and you're often entering the same values for change type and release notes. One thought I had was to auto-populate the release notes from the last deployment to simplify things for the user. 

In my experience, using the CLI is best because its commands never change, whereas a UI might change navigation and all instructions become invalid. This happened to me at Okta and we had to update over 200 blog posts. 

> Do we want to keep users on the CLI in these readmes, or state that deployment errors aren't shown on the CLI so it's better to continue in the UI?

It seems to me like you have to use the UI to do a release and install. Because of this conversation, I discovered there is a `foundry apps release` command. It doesn't look like there's a `foundry apps install` command. 

The reason I added instructions on how to find the app is because I myself couldn't find it when I first deployed an app. Yesterday, I noticed an "Open App" button in the success dialog after installing it. I found this very useful because I didn't even know a "Custom Apps" menu existing until last week. 

> I realize I've been reviewing your changes bit by bit instead of reading the whole readme, and I need to take some time to review the whole file.

You can see the whole file by going to my forked repo: 

https://github.com/mraible/foundry-sample-scalable-rtr/tree/readme-improvements#readme